src/status_im/utils/ethereum/erc20.cljs

```diff
@@ -57,4 +62,128 @@
 (defn allowance [web3 contract owner-address spender-address cb]
   (ethereum/call web3
                  (ethereum/call-params contract "allowance(address,address)" (ethereum/normalized-address owner-address) (ethereum/normalized-address spender-address))
-                 #(cb %1 (ethereum/hex->bignumber %2))))
\ No newline at end of file
+                 #(cb %1 (ethereum/hex->bignumber %2))))
+
+(defn- parse-json [s]
+  (try
+    (let [res (-> s
+                  js/JSON.parse
+                  (js->clj :keywordize-keys true))]
+      (if (and (:error res)
+               (= (:error res) ""))
+        {:result true}
+        res))
+    (catch :default e
+      {:error (.-message e)})))
+
+(defn- add-padding [address]
```

Comment:
`padded` sounds better? add sounds like mutable fn.

Reply:
I prefer to use verbs as function names. I don't think anyone would be confused by this name any more than they would be confused by say `sort` or `reverse` which also sound like they are mutating stuff if you take the name too literally.